index.html

```diff
@@ -0,0 +1,190 @@
+          type="button"
+          class="close"
+          data-dismiss="alert"
+          aria-label="Close"
+        >
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+    </div>
+
+    <div class="header">
+      <h1>Bot Controller</h1>
+      <div class="botSelector" style="width: 200px;">
+        <select id="botSelected" class="custom-select">
+          <option value="0">Select bot:</option>
+          <option value="1">Boomer</option>
+          <option value="2">nickbot</option>
+        </select>
+      </div>
+      <div class="toggle-dark">
+        <div class="toggle">
+          <input
+            type="button"
+            class="btn btn-primary"
+            value="toggle theme"
+            onclick="toggleTheme();"
+          />
+          <label for="toggle"></label>
+        </div>
+      </div>
+      <div class="status">
+        <div class="card bg-light">
+          <div class="card-header">
+            WebSocket:
+            <span id="serverStatus" class="badge badge-danger">Offline</span>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="flex-container">
+      <div class="controlCard">
+        <div class="card bg-light">
+          <h3 class="card-header">Controls</h3>
+          <div class="card-body">
+            <h6>Bot:</h6>
+            <button type="button" onclick="startBot()" class="btn btn-success">
+              start
+            </button>
+            <button type="button" onclick="stopBot()" class="btn btn-danger">
+              stop</button
+            ><br />
+            <h6 class="mt-4">Websocket:</h6>
+            <button
+              type="button"
+              onclick="startWebSocket()"
+              class="btn btn-success"
+            >
+              open
+            </button>
+            <button
+              type="button"
+              onclick="closeWebSocket()"
+              class="btn btn-outline-danger"
+            >
+              close
+            </button>
+          </div>
+        </div>
+      </div>
+      <div class="controlCard">
+        <div class="card bg-light">
+          <h3 class="card-header">Status</h3>
+          <div class="card-body">
+            <span class="dark-text">Coordinates:</span>
+
+            <span id="coords" class="badge badge-primary">0 0 0</span><br />
+
+            <span class="dark-text">Health: </span>
+            <span id="health" class="badge badge-danger">20</span><br />
+            <span class="dark-text">Hunger: </span>
+            <span id="hunger" class="badge badge-warning">20</span><br />
+          </div>
+        </div>
+      </div>
+      <div class="controlCard">
+        <div class="card bg-light">
+          <h3 class="card-header">Settings</h3>
+          <div class="card-body">
+            <form role="form">
+              <div class="form-group">
+                <label class="dark-text">Server address</label>
+                <input
```

Comment:
This `<input>` has the potential to conflict with the WASD keys used for movement, that could be dangerous - can we make sure those keys aren't triggered when in an input box?

Reply:
Will be fixed in https://github.com/eternum/minecraftBot/tree/toggles